webpack.config.js

```diff
@@ -37,7 +37,7 @@ let config = {
     filename: '[name].[chunkhash].js',
     chunkFilename: '[name].[chunkhash].js',
     path: PATHS.BUILD,
-    publicPath: '/'
+    publicPath: ''
```

Comment:
👍

Reply:
yea, that was fun to find. learned a bit about webpack, cool stuff.

Reply:
Nice!